site/content/en/docs/Installation/terraform.md

```diff
@@ -134,3 +135,73 @@ terraform destroy
 
 ### Reference
 Details on how you can authenticate your AKS terraform provider using official [instructions](https://www.terraform.io/docs/providers/azurerm/auth/service_principal_client_secret.html)
+
+
+{{% feature publishVersion="1.3.0" %}}
+# Installing the Agones as Terraform submodule on AWS EKS
+
+{{< alert title="Note" color="info" >}}
+AWS EKS Terraform configuration is not fully functional.
+`terraform destroy` leaves EKS cluster, so please remove EKS cluster and other objects, like VPC Security Groups manually.
+[Pull requests](https://github.com/googleforgames/agones/pulls) to finish the functionality are appreciated.
```

Comment:
Is this an issue with Terraform or Agones?

Reply:
Issue with Terraform providers, which probably could be fixed by updating `.tf` files contained in this PR. I will get an access to AWS soon, and will make a more thorough testing.

Reply:
The reason I ask is - by saying "we accept PRs" it sounds like it's an Agones problem, rather than a Terraform one.

Maybe we should say something like "This will be updated when the issue (link to issue) with Terraform is fully resolved" -- or something like that.

WDYT?

Reply:
Yes, that was copied, thought that this would be like a standard way.
I could not find exact issue now, there were many releases in between. 
I will remove this statement for now.